classes/Bili/FileIO.php

```diff
@@ -99,14 +102,22 @@ public static function html2pdf($strHtml, $strFilePrefix = "document", $arrParam
         $strInput = $strHtmlFile;
         $strOutput = $strPdfFile;
 
-        //*** Extra parameters.
-        $strParameters = (is_array($arrParameters)) ? implode(" ", $arrParameters) : "";
+        //*** Prepend extra parameters.
+        $strParameters = (is_array($arrParameters))
+            ? implode(" ", $arrParameters)
+            : "";
+
+        //*** Append extra parameters or define stdout and stderr
+        $strAppendParameters = (is_array($arrSettings['appendParameters']))
+            ? implode(" ", $arrSettings['appendParameters'])
+            : "";
 
```

Comment:
The above code assumes that the appendParameters value is always an array. Is that a good assumption? Could it be just a string as well? In my opinion it's even more likely to be a string than an array.